example/basic/main.go

```diff
@@ -78,6 +76,7 @@ func main() {
 
 	valuerecorderTwo := otel.Must(meter).NewFloat64ValueRecorder("ex.com.two")
 
+	ctx := context.Background()
```

Comment:
Why so?

Reply:
After merging with upstream [calling shutdown of the `SpanProcessor` directly is unneeded](https://github.com/open-telemetry/opentelemetry-go/pull/1227). This reverted a move to code that had no effective change given this direct call was removed and was extraneous to the function content of this PR.